### PR TITLE
✅   Balerion HotFix for Release Build Workflow

### LINKED ISSUE #121: Speed Up CI for Pull Request Workflow
The CI is pretty long for the PR workflow. This could be reduced by only having the Windows and Mac workflows run through the integration tests instead of through the entire test suite since this is already tested on Linux and on the release build.

### LINKED ISSUE #141: Dependency Graph fails on Release Build
This is related to a change in pydeps where the output is displayed by default. This causes the build to get stuck. 

This brings into play a different problem where the release order is not accurate as well. The pypi package should be the last thing completed. Both these issues should be fixed at the same time.

### LINKED ISSUE #142: Update version to v1.1.1
Update version number to 1.1.1 in setup.py

### PR DESCRIPTION
**Pull request by keithrfung**
### Issue

Fixes #121 
Fixes #141 
Fixes #142 

### Description
- Reduce MacOS and Windows to only run integration tests. 
- Add make file to run all integration tests
- Add simpler commands to make file
- Fix dependency graph command issue
- Increment release number

### Testing
- CI should pass. See example.
https://github.com/microsoft/electionguard-python/runs/951389340?check_suite_focus=true

### Checklist
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] 🤔 **CONSIDER** adding a unit test if your PR resolves an issue.
- [x] ✅ **DO** check open PR's to avoid duplicates.
- [x] ✅ **DO** keep pull requests small so they can be easily reviewed.
- [x] ✅ **DO** build locally before pushing.
- [x] ✅ **DO** make sure tests pass.
- [x] ✅ **DO** make sure any new changes are documented.
- [x] ✅ **DO** make sure not to introduce any compiler warnings.
- [x] ❌**AVOID** breaking the continuous integration build.
- [x] ❌**AVOID** making significant changes to the overall architecture.

💚Thank you!

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
+atomicwrites = "*"
```
Comment (keithrfung) [inline Pipfile:7]: This is required for windows tests. `pytest` requires it but it is not installed by default.

File: Makefile [hunk 2]
```diff
@@ -115,8 +119,18 @@ docs-deploy:
...
+	mkdocs gh-deploy --force
+
 dependency-graph:
-	pipenv run pydeps --max-bacon 2 -o dependency-graph.svg src/electionguard
+	pipenv run pydeps --noshow --max-bacon 2 -o dependency-graph.svg src/electionguard
+
+dependency-graph-ci:
+	sudo apt install graphviz
+	pip install pydeps
+	pydeps --noshow --max-bacon 2 -o dependency-graph.svg src/electionguard
```
Comment (keithrfung) [inline Makefile:133]: The lack of this `--no-show` is what caused the build to fail.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -118,6 +117,13 @@ jobs:
           release_name: Release ${{ env.PACKAGE_VERSION }}
           draft: false
           prerelease: false
+      - name: Dependency Graph
```
Comment (keithrfung) [inline .github/workflows/release.yml:120]: Moved the order of this up so it wouldn't block the rest of the build after pypi is pushed.

**Comment by keithrfung** [comment]
Please review.

**Comment by rc-ms** [comment]
This looks great!  Thank you for your patience Keith!

**Pull request merged**